### PR TITLE
Update Viewport on resize in all of the GL tutorials.

### LINKED ISSUE #1964: Camera Tutorial Demo doesn't handle window resizes.
# Summary
Camera Tutorial Demo doesn't handle window resizes.

# Steps to reproduce
image

# Comments
My own code has this issue.  Not sure how to solve this, as your tutorials have the same issue.

### PR DESCRIPTION
**Pull request by otac0n**
Fixes #1964

### CODE REVIEW

**Comment by otac0n** [comment]
@dotnet-policy-service agree

File: examples/CSharp/OpenGL Tutorials/Tutorial 2.2 - Camera/Program.cs [hunk 1]
```diff
@@ -184,6 +185,11 @@ private static unsafe void OnRender(double deltaTime)
             Gl.DrawArrays(PrimitiveType.Triangles, 0, 36);
         }
 
+        private static void OnFramebufferResize(Vector2D<int> newSize)
```
Comment (Perksey) [inline examples/CSharp/OpenGL Tutorials/Tutorial 2.2 - Camera/Program.cs:188]: Don't these tutorials have size-based uniforms to update as well? (search for usage of the Width and Height constants)

**Review by Perksey — CHANGES_REQUESTED** [review]
Amazing, thank you so much! Just one small comment, it's hard to find anything wrong with these changes haha.

File: examples/CSharp/OpenGL Tutorials/Tutorial 2.2 - Camera/Program.cs (hunk 1 above)
Comment (otac0n) [reply examples/CSharp/OpenGL Tutorials/Tutorial 2.2 - Camera/Program.cs:187]: Yep, fixing it.

File: examples/CSharp/OpenGL Tutorials/Tutorial 3.2 - Diffuse Lighting/Program.cs [hunk 2]
```diff
@@ -170,6 +170,11 @@ private static unsafe void OnRender(double deltaTime)
             RenderLampCube();
         }
 
+        private static void OnFramebufferResize(Vector2D<int> newSize)
```
Comment (otac0n) [inline examples/CSharp/OpenGL Tutorials/Tutorial 3.2 - Diffuse Lighting/Program.cs:172]: Not referenced :(

**Comment by otac0n** [comment]
I have run every demo, although to get them to build I referenced the live NuGet packages (rather than project references which wouldn't build at the moment.)

**Review by otac0n — COMMENTED** [review]
Fixed an extra issue with 4.1, let me know if it should be moved to its own PR.

**Review by Perksey — APPROVED** [review]
Awesome, thanks!

**Pull request merged**